specs/MultiProfile.md

```diff
@@ -29,6 +29,21 @@ Users can use this interface to get the cookie manager, which is shared by all W
 with this profile. The cookie manager got from profile (CoreWebView2.Profile.CookieManager) is the
 same as that got from CoreWebView2 (CoreWebView2.CookieManager).
 
+Currently, we already have **CoreWebView2Settings** interface to manage password-autosave and
+general-autofill, but it will work not immediately but after the next navigation, and it can only
```

Comment:
Should avoid using the verb "work". The feature "works" in that it does change the autofill setting on the next navigation.

Recommend using "take effect" throughout, rather than "work".

Reply:
Thanks, replaced "work" with "take effect".